risc0/circuit/rv32im/src/prove/emu/exec/mod.rs

```diff
@@ -88,8 +88,14 @@ pub trait SyscallContext {
         Ok(region)
     }
 
+    /// Load a page from memory at the specified page index.
```

Comment:
Can we document what the difference is between peek_page and peek_region and why we'd use one over the other?

Reply:
I added to the `peek_region`:

> A region may span multiple pages.